modules/ROOT/pages/using-api-alerts.adoc

```diff
@@ -51,9 +47,13 @@ When you set an alert, you select users in your Business Group who can receive e
 +
 `Your API Version, jsonplaceholderapi - 1.0.development, is no longer in an alert state. The number of policy violations was not greater than 1 in the last 1 consecutive periods of 1 minutes.`
 
-After an alert fires, and API Manager sends the first set of two notification emails, and stops listening for alerts until the next alert period begins. This technique prevents repetitive notification emails.
+After an alert is triggered, API Manager sends the first set of two notification emails, and stops listening for alerts until the next alert period begins. This technique prevents repetitive notification emails.
+
+== Contracts Alerts
 
-== Alert Notification Categories
+API Manager automatically notifies users when actions related to contracts occur. For instance, when a client application requests access to an API instance, API Manager sends a notification indicating that the request access is pending (or that the request access has been auto-approved, depending on the configuration).
```

Comment:
Optional or save for a later revision: 2nd sentence could be easier to read if shorter or structured differently. If "pending" and "auto-approved" are the only two possible statuses, you could make a bullet list. Maybe you could simplify the details in the example? Something like 

"...sends a notification with the status of the access request."

Reply:
Cool, good suggestion. I split up that long sentence. Thanks!